packages/@aws-cdk/aws-ecs/lib/base/from-service-attributes.ts

```diff
@@ -27,7 +27,15 @@ export interface ServiceAttributes {
   readonly serviceName?: string;
 }
 
-export function fromServiceAtrributes(scope: Construct, id: string, attrs: ServiceAttributes): IBaseService {
+export function fromServiceAttributes(scope: Construct, id: string, attrs: ServiceAttributes): IBaseService {
+  return fromServiceAtrributes(scope, id, attrs);
+}
+
+/**
+ * DEPRECATED
+ * @deprecated use fromServiceAttributes()
```

Comment:
I don't think we need to actually deprecate this. It's not exposed to the user, so this shouldn't be a breaking change we have to keep around. Can we only have the `fromServiceAttributes()` function?

Reply:
Thanks, I did.